docs/step_by_step_guide.md

```diff
@@ -0,0 +1,646 @@
+# Useful Aliases
+Setting some aliases may be useful in running the model. You can edit your bash settings file to add aliases. Run `source` on that file to start using your new aliases. Examples:
+- Chrysalis – `source ~/.bashrc`
+- Compy – `source ~/.bash_profile`
+- Perlmutter – `source ~/.bash_profile.ext`
```

Comment:
These bullets show up fine on the GitHub rendering but all in one line on the actual rendering...

Reply:
This problem recurs throughout the file.

Reply:
Yes that would be great. Thanks @mahf708! (I still think the docs on developing our docs should be updated to reflect that we have some differences from standard GitHub markdown.)